CHANGELOG.txt

```diff
@@ -1,5 +1,136 @@
 == Changelog ==
 
+= 3.7.0 - 2019-08-12 =
```

Comment:
I believe it is better to keep `3.7.0` out of the changelog until it is released. That because we point people to the version of this file in the master branch from the readme.txt file to see the full changelog and it could be confusing for users to see a version that hasn't been released yet: https://github.com/woocommerce/woocommerce/blob/fa9dfa64e20ff60f3e24bb40b63f036da9d2bed3/readme.txt#L284